### PR TITLE
ENH: Updated to latest pysat custom method signature

### PR DESCRIPTION
**Pull request by rstoneback**
# Description

Addresses #(issue)

Updates package for pysat breaking changes in the custom kernel functionality.

## Type of change

Please delete options that are not relevant.

- Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Tested locally in python 3 on macOS 10.15.4

# Checklist:

- [x] Make sure you are merging into the ``develop`` (not ``main``) branch
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules
- [x] Add a note to ``CHANGELOG.md``, summarizing the changes

### CODE REVIEW

**Comment by rstoneback** [comment]
Tests aren't going to pass until pull pysat/pysat#616 is merged.

**Review by jklenzing — APPROVED** [review]
Passing locally.  Approved pending merge of pysat/pysat#616

**Pull request merged**
